internal/cephfs/controllerserver.go

```diff
@@ -199,6 +234,11 @@ func (cs *ControllerServer) CreateVolume(
 		defer parentVol.Destroy()
 	}
 
+	err = checkValidCreateVolumeRequest(volOptions, parentVol, pvID, sID)
+	if err != nil {
+		return nil, err
```

Comment:
This just returns the error, instead of converting it to a gRPC error. The `checkValidCreateVolumeRequest()` function takes care of that, but it would be cleaner to do it here.

Ideally top-level gRPC functions are the only ones that return gRPC errors, all other functions should just return plain errors so that they can be re-used outside of the direct gRPC flow.